### PR TITLE
feat(fa): On status page updating timeline

### PR DESCRIPTION
**Pull request by MargretFinnboga**
# ...

https://app.asana.com/0/1201365172053613/1201380246716004

## What
- Updating new design for timeline 
- Fixing button on table in veita

## Why

- For mobile and better user experience 

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

File: apps/financial-aid/web-osk/src/components/Timeline/timeline.tsx [hunk 1]
```diff
@@ -1,64 +1,78 @@
...
-        {
-          type: 'SUB_SECTION',
-          name:
-            'Umsókn verður samþykkt eða synjuð og umsækjandi látinn vita um niðurstöðuna',
-        },
-      ],
+      text:
+        'Umsókn verður samþykkt eða synjuð og umsækjandi látinn vita um niðurstöðuna',
+      state: [ApplicationState.REJECTED, ApplicationState.APPROVED],
+      date: format(new Date(modified), 'dd/MM/yyyy HH:MM'),
```
Comment (Rikkiabb) [inline apps/financial-aid/web-osk/src/components/Timeline/timeline.tsx:38]: Nice

File: apps/financial-aid/web-osk/src/components/Timeline/timeline.tsx [hunk 2]
```diff
@@ -1,64 +1,78 @@
...
+              [`${styles.lineDown}`]: index !== sections.length - 1,
+            })}
+          >
+            <Box paddingLeft={3}>
+              <Text variant="h5">{item.name}</Text>
+              <Text marginBottom={2}>{item.text}</Text>
+
+              <Text variant="small" color="dark300" marginBottom={5}>
+                {(index === 0 || index === activeState) && item.date}
+              </Text>
```
Comment (Rikkiabb) [inline apps/financial-aid/web-osk/src/components/Timeline/timeline.tsx:71]: Do you want to do the boolean check on line 69 instead so we don't have the bottom margin if this is null?

**Review by Rikkiabb — COMMENTED** [review]
One ❓

File: apps/financial-aid/web-osk/src/components/Timeline/timeline.tsx (hunk 2 above)
Comment (MargretFinnboga) [reply apps/financial-aid/web-osk/src/components/Timeline/timeline.tsx:71]: No the margin is nice here is example, so the spacing is always the same 
without margin:
Screenshot 2021-11-19 at 15 46 50

with margin: 
 
Screenshot 2021-11-19 at 15 47 07

I think the same space is nicer, what do you think? 👨‍🎨

Comment (Rikkiabb) [reply apps/financial-aid/web-osk/src/components/Timeline/timeline.tsx:71]: Agree, just looked strange in code.

**Review by Rikkiabb — APPROVED** [review]

**Pull request merged**